common/app/assets/assets.scala

```diff
@@ -152,6 +152,7 @@ class Assets(base: String, assetMapPath: String = "assets/assets.map") extends L
 
      val curl: String = RelativePathEscaper.escapeLeadingDotPaths(inlineJs("assets/curl-domReady.js"))
      val omnitureJs: String = inlineJs("assets/vendor/omniture.js")
+     val reactPath: String = "https://cdnjs.cloudflare.com/ajax/libs/react/0.13.2/react.min.js"
```

Comment:
using a CDN for such a popular lib massively increases the chances we'd hit the http cache. agree we should be able to fall back to our own if it fails, but this seems a particularly strong case for a CDN to me (we don't want to bundle it, it's very popular)?